document/src/vespa/document/serialization/vespadocumentserializer.cpp

```diff
@@ -368,7 +368,7 @@ VespaDocumentSerializer::write(const WeightedSetFieldValue &value) {
 void
 VespaDocumentSerializer::write(const TensorFieldValue &value) {
     vespalib::nbostream tmpStream;
-    auto &tensor = value.getAsTensorPtr();
+    auto tensor = value.getAsTensorPtr();
```

Comment:
Do we want to explicitly signal that these are raw pointers with `auto*` for such variables?

Reply:
+1 for auto* here